examples/custom/01-parameter-passing/demo.fth

```diff
@@ -0,0 +1,17 @@
+\ f4711 is a clear indicator that compilation of custom functions was successful
+." f4711( 0, 1, 10, 100, 1000 ) = ( "
+   0 f4711 . ." , "
+   1 f4711 . ." , "
+  10 f4711 . ." , "
+ 100 f4711 . ." , "
+1000 f4711 . ." )"
+CR
+
+\ example of passing passing strings from PForth to custom C code
```

Comment:
Still "passing passing".

Reply:
Sorry, did not realize the double word.

Done.